### PR TITLE
event hwdb fix for Hp Pavilion G6

### PR DESCRIPTION
**Pull request by SparrowOchon**
Synaptics touchpad edge fix for HP Pavilion G6

### CODE REVIEW

File: hwdb/60-evdev.hwdb [hunk 1]
```diff
@@ -257,6 +257,13 @@ evdev:name:SynPS/2 Synaptics TouchPad*:dmi:*svnHewlett-Packard:pnHPPaviliondm4*
  EVDEV_ABS_35=1360:5563:47
  EVDEV_ABS_36=1269:4618:61
 
+# HP Pavilion g6 
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnHewlett-Packard:pnHPPaviliong6*
+ EVDEV_ABS_00=1284:5696:88
+ EVDEV_ABS_01=1287:4838:39
+ EVDEV_ABS_35=1284:5696:88
+ EVDEV_ABS_36=1287:4838:39
+
```
Comment (yuwata) [inline hwdb/60-evdev.hwdb]: Please drop the trailing whitespace.

Comment (SparrowOchon) [reply hwdb/60-evdev.hwdb]: Removed

Comment (yuwata) [reply hwdb/60-evdev.hwdb]: Thanks.

**Comment by jrodr236** [comment]
This change causes a problem.

Now, touchpad vertical sensitivity is much higher than horizontal sensitivity in HP Pavilion G6.

See more details here: https://bugs.launchpad.net/ubuntu/+source/libinput/+bug/1825499

**Comment by keszybz** [comment]
Hmm, the machine in Ubuntu bug report has:

dmi.bios.date: 05/29/2013
dmi.bios.vendor: Insyde
dmi.bios.version: F.25
dmi.board.asset.tag: Type2 - Board Asset Tag
dmi.board.name: 183E
dmi.board.vendor: Hewlett-Packard
dmi.board.version: 56.32
dmi.chassis.type: 10
dmi.chassis.vendor: Hewlett-Packard
dmi.chassis.version: Chassis Version
dmi.modalias: dmi:bvnInsyde:bvrF.25:bd05/29/2013:svnHewlett-Packard:pnHPPaviliong6NotebookPC:pvr0885120000005D10000600000:rvnHewlett-Packard:rn183E:rvr56.32:cvnHewlett-Packard:ct10:cvrChassisVersion:
dmi.product.family: 103C_5335KV G=N L=CON B=HP S=PAV X=Null
dmi.product.name: HP Pavilion g6 Notebook PC
dmi.product.sku: D4Z91EA#ABV
dmi.product.version: 0885120000005D10000600000
dmi.sys.vendor: Hewlett-Packard

@SparrowOchon it looks like your patch needs to be amended to be **not** match the above. We need to figure out how they can be distinguished. Can yo post the full dmi for  your machine too?

**Comment by keszybz** [comment]
Let's continue the disucssion in #13116.

**Pull request merged**